doc-surrealdb_versioned_docs/version-1.x/surrealql/statements/update.mdx

```diff
@@ -47,15 +47,35 @@ UPDATE person:100 SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', '
 -- Update or create a record with a specific string id
 UPDATE person:tobie SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
 
--- Update just a single record
 -- Using the ONLY keyword, just an object for the record in question will be returned.
 -- This, instead of an array with a single object.
 UPDATE ONLY person:tobie SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
 ```
+
+As Record IDs work like pointers, an update to a non-existing record will create the record. Note the difference in behaviour between an update on all records and an update on a single specified record ID:
```

Comment:
I wonder if it might be more beneficial to rewrite this page to highlight `UPDATE` as part of the Data definition language in SurrealDB. The insert statement will do the same.